packages/localization/src/i18n/messages/en-gb.ts

```diff
@@ -116,6 +116,8 @@ export const EN_GB_MESSAGES: Messages = {
     "task.paymentSuccessTitle": "Success, you have paid.",
     "overviewpage.title": "Welcome {userName}",
     "overview.subTitle": "Authorised for {volmachtgever}",
+    "overviewpage.paragraph":
+      "In MijnDenHaag you can arrange your personal affairs yourself when it suits you. For example, you can pay your bills and see when your application is ready.",
```

Comment:
Ik zou deze tekst iets generieker maken hier, en overschrijven in onze Den Haag repo. Aangezien dit de generieke libraries zijn.

Reply:
aangepast